NCI-GDC/portal-ui
# ✏️ PRTL-1353, PRTL-1354 use gene+aachange instead of mutation id for survival legend， Sort facet buckets by doc_count desc

![image](https://user-images.githubusercontent.com/743976/27490415-255669f0-580c-11e7-8ae1-778325b1153d.png)

Test Instructions:
Open this page
http://deploybot.gdc.cheapsteak.net:32845/exploration?facetTab=mutations&searchTableTab=mutations

And verify that facet buckets are sorted by descending order, and clicking on a Survival Analysis button would result in the overall survival plot showing a legend with the Gene Symbol + AA Change
